docs/api_reference.rst

```diff
@@ -0,0 +1,12 @@
+api_reference
+===============
+
+.. automodule:: fileinfo
```

Comment:
```suggestion
.. automodule:: surfactant.fileinfo
```

Needs to specify the full import path to the fileinfo module (`surfactant.fileinfo`) -- I think this is again an instance of Linux being a bit pickier.